### DISCUSSION TITLE
Meson for building OP-TEE libraries

### DISCUSSION THREAD

**Discussion by lindrix** [mesonbuild/meson#14404] [General]
Hi.

Is there meson support  to build and add libraries in Op-TEE? How to use meson for these operations?

Thank you for attention.

**Comment by eli-schwartz** [comment]
Have you tried setting up a meson project using the $CC for the right compiler? In general, shouldn't have limitations here except to the extent that the compiler is known.

See e.g. https://mesonbuild.com/Reference-tables.html#compiler-ids
